clients/admin-ui/src/features/datastore-connections/system_portal_config/forms/ConnectorParametersForm.tsx

```diff
@@ -235,18 +235,23 @@ export const ConnectorParametersForm = ({
         connectionConfig.enabled_actions || []
       ).map((action) => action.toString());
 
-      // @ts-ignore
-      initialValues.secrets = connectionConfig.secrets ?? {};
+      initialValues.secrets = connectionConfig.secrets
+        ? { ...connectionConfig.secrets }
+        : {};
```

Comment:
the frontend was exploding when trying to process the fides dataset reference. I suspect the underlying issue was there from before and we simply weren't using this? anyways this is a quick & dirty fix (same with the fix below) to get it to work for now, I'm happy to create a ticket to address this more thoroughly later

Reply:
Can you be more specific about what was exploding, I'm wondering if it is pydantic related?